dataset.py

```diff
@@ -333,3 +308,48 @@ def tensor2im(image_tensor, imtype=np.uint8):
 def save_image(image_numpy, image_path):
     image_pil = Image.fromarray(image_numpy)
     image_pil.save(image_path)
+
+if __name__ == '__main__':
+    """
+    Main method
+    """
+    trainX, trainY = readData(TRAIN_FILENAME) # Training data
+    testX, testY = readData(TEST_FILENAME) # Testing data
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-e", "--epochs", dest="epochs", default=1000,
+                        type=int, help="Number of epochs")
+    parser.add_argument("-lr", "--learningrate", dest="learningRate", default=0.07,
+                        type=float, help="Learning rate or step size")
+    parser.add_argument("-bs", "--batchSize", dest="batchSize", default=10,
+                        type=int, help="Number of sample in mini-batches")
+    parser.add_argument("-r", "--regStrength", dest="regStrength", default=0.001,
+                        type=float, help="L2 weight decay regularization lambda value")
+    parser.add_argument("-m", "--momentum", dest="momentum", default=0.05,
+                        type=float, help="A momentum value")
+
+    args = parser.parse_args()
+
+    print(
```

Comment:
remove Print statement before deploying model to production

Reply:
removed!